ONSdigital/dp-filter-api
# Feature/update data model

### What

Updated the data fields that get stored in postgres; replace `dataset`, `edition` and `version` with `dataset_filter_id` which is the equivalent of a `instance_id` and represents a unique dataset edition and version. Removal of json body when creating a filter job, this is not needed.

Update swagger specs to be inline with implementation of API service.

### How to review

* Make sure tests still pass.
* Check for consistent naming of variables.
* Check swagger spec is as expected and there are no spelling/grammar mistakes.

### Who can review

Team B
